src/Symfony/Component/Form/Extension/Core/Type/MoneyType.php

```diff
@@ -27,12 +29,15 @@ class MoneyType extends AbstractType
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        // Values used in HTML5 number inputs should be formatted as in "1234.5", ie. 'en' format without grouping,
+        // according to https://www.w3.org/TR/html51/sec-forms.html#date-time-and-number-formats
         $builder
             ->addViewTransformer(new MoneyToLocalizedStringTransformer(
                 $options['scale'],
-                $options['grouping'],
+                $options['html5'] ? $options['grouping'] : false,
```

Comment:
Shouldn't this be the other way around (i.e. `!$options['html5'] ? $options['grouping'] : false`)?

Reply:
On the other hand: We know that `$options['grouping]` is `false` when `html5` is enabled. So we do not need to change anything here at all, do we?

Reply:
That's correct, that's redundant with the normalizer set line 86 (to have a nice error message).